dependency-check-maven-config.xml

```diff
@@ -0,0 +1,485 @@
+            when packaging war
+        </notes>
+        <cve>CVE-2014-0050</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2016-1000031: Applies to commons-fileupload-1.2.1, suppressing because the
+            vulnerable class DiskFileItem is not used in the project
+        </notes>
+        <cve>CVE-2016-1000031</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        false positive the effected camel version is 2.12 this uses a later version and does use the XSLT component
+      file name: proxy-camel-servlet-2.9.0-SNAPSHOT.jar
+      ]]></notes>
+        <cve>CVE-2014-0002</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+        false positive the effected camel version is 2.12 this uses a later version and does use the XSLT component
+   file name: proxy-camel-servlet-2.9.0-SNAPSHOT.jar
+   ]]></notes>
+        <cve>CVE-2014-0003</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        this CVE is generating a lot of false positives it should only include jackson-dataformat-xml jar
+         but it is catching all jackson dependencies unfortunately I can't fix it so if we start depending
+          on a vulnerable version of jackson-dataformat-xml this will still suppress it.
+   ]]></notes>
+        <cve>CVE-2016-3720</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+   file name: commons-beanutils-1.8.3.jar
+   shiro-core has a dependency on this but it doesn’t expose commons-beanutils to user input so it wouldn't pose a risk like the struts library that is called out in the CVE
+   ]]></notes>
+        <sha1>686EF3410BCF4AB8CE7FD0B899E832AABA5FACF7</sha1>
+        <cve>CVE-2014-0114</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+   file name: ffmpeg-3.1.1_1-bin.zip: ffmpeg.exe
+   Reported CVE's are vulnerabilities in earlier versions of FFmpeg
+   ]]></notes>
+        <sha1>6EECC43A0883EF7C2A8E4BAF3AD45E24D8119ECC</sha1>
+        <cpe>cpe:/a:ffmpeg:ffmpeg:-</cpe>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        false positive CVE is unrelated
+   file name: platform-filter-delegate-*.jar
+   ]]></notes>
+        <cve>CVE-2005-0861</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        false positive CVE is unrelated
+   file name: nagasena-0000.0002.0049.0.jar
+   ]]></notes>
+        <sha1>8821DF6D1BC22FBA8F608021F4F27698CF3FD91F</sha1>
+        <cve>CVE-2014-9389</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        false positive CVE is unrelated
+   file name: org.apache.servicemix.bundles.not-yet-commons-ssl
+   ]]></notes>
+        <cve>CVE-2004-0009</cve>
+    </suppress>
+
+    <!-- these are the geowebcache vulnerabilities it is not installed by default and it is only experimental these security issues would need to be resolved before geowebcache can be installed in a production enviroment-->
```

Comment:
Should we put this comment in the notes field of each of the geowebache suppressions?